src/Plugin/EncryptionMethod/McryptAES256Encryption.php

```diff
@@ -12,7 +12,8 @@
  * @EncryptionMethod(
  *   id = "mcrypt_aes_256",
  *   title = @Translation("Mcrypt AES 256"),
- *   description = "This uses PHPs mcrypt extension and <a href='http://en.wikipedia.org/wiki/Advanced_Encryption_Standard'>AES-256</a>."
+ *   description = "This uses PHPs mcrypt extension and <a href='http://en.wikipedia.org/wiki/Advanced_Encryption_Standard'>AES-256</a>.",
```

Comment:
This will be handled with https://github.com/d8-contrib-modules/encrypt/issues/68.
Didn't want to make this PR too big :)
I'll work on removing MCrypt Encryption once we're sure we want to give it a new home, or abandon it completely.